packages/account/src/Containers/routes.tsx

```diff
@@ -16,7 +16,10 @@ const Routes = observer(() => {
 
     return (
         <ThemeProvider theme={is_dark_mode_on ? 'dark' : 'light'}>
-            <BinaryRoutes is_logged_in={is_logged_in} is_logging_in={is_logging_in} />
+            <SnackbarProvider>
```

Comment:
why not we use standalone snackbar comonent?

https://quill-ui.pages.dev/?path=/docs/components-snackbar-standalone--documentation

Reply:
I tried to use standalone snackbar component but the design is not as expected that's why i added the SnackbarProvider here